docs/releaseNotes.yml

```diff
@@ -32,6 +32,22 @@
 
 changelog: https://github.com/emissary-ingress/emissary/blob/$branch$/CHANGELOG.md
 items:
+  - version: 3.1.0
+    date: 'TBD'
+    notes:
+      - title: Register secrets used with an API Key filter.
+        type: change
+        body: >-
+          Emissary will register a secret into its internal machinery if it is used by
+          an API Key filter, since the value has to be read at filter creation.
+
+    edgeStackNotes:
```

Comment:
I don't think the release notes are required. That said if we do have edgestack relnotes then putting thme under `edgeStackNotes:` is appropriate although not really done anymore since I don't think the docs render this section anymore.